### PR TITLE
[OT196-66] Endpoint Create Activity

### PR DESCRIPTION
**Pull request by ezequielmgonzalez**
https://alkemy-labs.atlassian.net/browse/OT196-66

### Summary
- Activities router in app
- New endpoint: POST /activities
- Activity validator for new activities
- New controller to create activity

### Evidence

If the user does not have an admin role:
image

If name or content is missing in the new activity:
image

If the user is an admin and the new activity has name and content:
image

### CODE REVIEW

File: controllers/activitiesController.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        return res.status(200).send({
+          ok: true,
+          message: "Activity created.",
+          newActivity: newActivity,
+        });
+    } catch (e) {
+      // Response with error on fail request
+      return res.status(400).send({
+        ok: false,
+        message: "Error creating contact.",
```
Comment (GauntStrelok) [inline controllers/activitiesController.js:28]: Mejorar el msj de error

File: controllers/activitiesController.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+// Add Activity's Model
+const { Activity } = require("../models/index");
+
+exports.createActivity = async (req, res, next) => {
+  try {
```
Comment (GauntStrelok) [inline controllers/activitiesController.js:5]: Para que hay un try aca?

**Review by GauntStrelok — CHANGES_REQUESTED** [review]

**Comment by GauntStrelok** [comment]
Hacer pull de main para arreglar conflictos y revisar los comentarios enviados

File: controllers/activitiesController.js (hunk 2 above)
Comment (ezequielmgonzalez) [reply controllers/activitiesController.js:5]: La realidad es que se me escapó, perdón. Ya lo corrijo. Gracias Pablo!

**Pull request merged**